project/settings/defaults.py

```diff
@@ -226,7 +233,7 @@
 
 
 # Configuration for cross-site requests
-CORS_ORIGIN_ALLOW_ALL = True  # Needs to be changed before production!
+CORS_ORIGIN_ALLOW_ALL = True  # TODO Needs to be changed before production!
```

Comment:
No worries. I just always do a TODO search before prod launch and if not fix them at least decide they're not terrifying.